### DISCUSSION TITLE
Question about which interface to use in external mode?

### DISCUSSION THREAD

**Discussion by GurdipS5** [loxilb-io/loxilb#881] [Q&A]
Hi,
I am setting up loxilb in external mode. The steps says this:

## Set underlying interface of the VM/cluster-node to promisc mode for mac-vlan to work
sudo ifconfig eth1 promisc

However, on a vm with docker installed, there are several network interfaces:

![image](https://github.com/user-attachments/assets/96cef252-a9d9-409b-a336-8ae336f23b6b)

Of the first two, which NIC do I want to use?

**Comment by TrekkieCoder** [comment]
I believe in this case, you have to use ```ens160``` as the interface.  You can also run loxilb with net=host option, in which case you do not need to use mac-vlan or other interfaces :

```
sudo docker run -u root --cap-add SYS_ADMIN --net=host --restart unless-stopped --privileged -dit -v /dev/log:/dev/log --name loxilb ghcr.io/loxilb-io/loxilb:latest
```

With the above, all the host's interfaces will be visible to loxilb and there would be no need to created additional interfaces.

**Comment by GurdipS5** [comment]
Yeah ens160 makes sense. That is the original IP/network of the VM, pre Docker.

There is this step now:

sudo docker network create -d macvlan -o parent=eth1 --subnet 192.168.82.0/24   --gateway 192.168.82.1 --aux-address 'host=192.168.82.252' llbnet

I assume this should be the main subnet and gateway. So the vm is in 10.0.0.0/15 with a gateway in that range, I assume I should be using those details? And host would be the server's IP.

Thanks
